drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -113,6 +113,7 @@ struct RenderDataGLES3 {
 	int reflection_probe_pass = 0;
 
 	float lod_distance_multiplier = 0.0;
+	Plane lod_camera_plane = Plane();
```

Comment:
```suggestion
	Plane lod_camera_plane;
```
This is already defaulted